README.md

```diff
@@ -130,6 +131,57 @@ All examples are in [docs/examples.md](docs/examples.md) and are generated from
 
 Also take a look at [cypress-io/cypress-example-circleci-orb](https://github.com/cypress-io/cypress-example-circleci-orb) and [cypress-io/cypress-example-kitchensink](https://github.com/cypress-io/cypress-example-kitchensink/pull/148/files).
 
+## Naming
+
+When importing this orb, we suggest using local name "cypress" for consistency.
+
+```yaml
+version: 2.1
+orbs:
+  #          ↱ official orb name in the registry (org + name)
+  cypress: cypress-io/cypress@1
+  #  ↳ your local name for the imported orb
+workflows:
+  build:
+    jobs:
+      #   ↱ local orb name
+      - cypress/run
+      #          ↳ job "run" defined in the orb
+```
+
+You can of course use another local name
+
+```yaml
+version: 2.1
+orbs:
+  #             ↱ official orb name in the registry (org + name)
+  e2eCypress: cypress-io/cypress@1
+  #  ↳ your local name for the imported orb
+workflows:
+  build:
+    jobs:
+      #   ↱ local orb name
+      - e2eCypress/run
+      #             ↳ job "run" defined in the orb
+```
+
+But we suggest using local name `cypress` and instead giving explicit name to each job.
```

Comment:
should it be "and instead giving `an` explicit name to each job"? I'm not sure if we have a set style for how concise to make the text. Will approve, fine with it either way. Rest looks great! 👍

Reply:
yeah, sounds a little wrong